webapp/portlet/src/main/webapp/user-setting-security/components/UserSettingSecurity.vue

```diff
@@ -49,6 +52,13 @@ export default {
     });
     document.addEventListener('showSettingsApps', () => this.displayed = true);
     this.skeleton = false;
+    getCurrentUser().then(data => {
+      if(data && data.isInternal){
```

Comment:
this.canChangePassword= data && (data.isInternal || exo.portal.allow.change.external.password == true) ;